docs/monitors/collectd-df.md

```diff
@@ -38,28 +44,28 @@ Metrics that are categorized as
 (*default*) are ***in bold and italics*** in the list below.
 
 
- - ***`df_complex.free`*** (*gauge*)<br>    Free disk space in bytes
- - `df_complex.reserved` (*gauge*)<br>    Reserved disk space in bytes
- - ***`df_complex.used`*** (*gauge*)<br>    Used disk space in bytes
- - `percent_inodes.free` (*gauge*)<br>    Free inodes on the file system, expressed as a percentage.
- - `percent_inodes.reserved` (*gauge*)<br>    Reserved inodes on the file system, expressed as a percentage.
- - `percent_inodes.used` (*gauge*)<br>    Used inodes on the file system, expressed as a percentage.
+ - ***`df_complex.free`*** (*gauge*)<br>    This metric measures free disk space in bytes on this file system.
```

Comment:
I realize you just took these from the integrations repo but I just want to point out it's redundant to say "This metric measures..." but every single metric description. Maybe whoever wrote it chose to do it so it was a complete sentence but I think it adds a lot of noise.

Reply:
Stripped off the leading "This metric ".

Reply:
It kind of needs to be a sentence if we are going to put a period at the end of it, esp in the cases where there are multiple sentences.

Reply:
"Measures free inodes as a percentage of total inodes in the file system." isn't an actual sentence though right? I think it sounds fine for these though since it reads as  "df_complex.free measures free inodes as a percentage of total inodes in the file system"

Reply:
Yes, it is a sentence with an implicit subject of "This".